### ISSUE TITLE
My 5070 series graphics card wasn't working properly. I had AI help me troubleshoot and modify part of the source code, and it ran normally. I'd like to ask if this is a good idea.

### ISSUE THREAD

**Issue by dy-ys** [lihaoyun6/ComfyUI-FlashVSR_Ultra_Fast#67]
filePath ： /custom_nodes/ComfyUI-FlashVSR_Ultra_Fast/src/models/ wan_video_dit.py

Replace the function def flash_attention with：

def flash_attention(q: torch.Tensor, k: torch.Tensor, v: torch.Tensor, num_heads: int, compatibility_mode=False, attention_mask=None, return_KV=False):
    # -----------------------------------------------------------
    # 修正版 V2 (适用于 RTX 5070 / PyTorch 2.7):
    # 1. 强制使用 PyTorch 原生 SDPA，绕过 Triton 编译错误。
    # 2. 修复了 Cross Attention 中 Q 和 K 长度不一致导致的 shape 报错。
    # -----------------------------------------------------------
    
    # 1. 获取 Q 的维度 [Batch, SeqLen_Q, Dim]
    b, s_q, c = q.shape
    head_dim = c // num_heads

    # 2. 获取 K 的维度 [Batch, SeqLen_K, Dim]
    # 关键点：在 Cross Attention 中，s_k (文本长度 512) 不等于 s_q (视频长度 6144)
    # 我们必须使用 k 自己的长度参数
    _, s_k, _ = k.shape

    # 3. 转换 Q -> [Batch, NumHeads, SeqLen_Q, HeadDim]
    q = q.view(b, s_q, num_heads, head_dim).transpose(1, 2)
    
    # 4. 转换 K, V -> [Batch, NumHeads, SeqLen_K, HeadDim]
    # 这里必须使用 s_k，而不是 s_q
    k = k.view(b, s_k, num_heads, head_dim).transpose(1, 2)
    v = v.view(b, s_k, num_heads, head_dim).transpose(1, 2)

    # 5. 执行原生注意力计算
    # PyTorch FlashAttention 会自动处理 s_q 和 s_k 不一致的情况
    # 我们忽略传入的 attention_mask，因为那是为稀疏注意力设计的，全量计算不需要
    x = F.scaled_dot_product_attention(q, k, v, attn_mask=None, is_causal=False)

    # 6. 还原结果形状 -> [Batch, SeqLen_Q, Dim]
    x = x.transpose(1, 2).contiguous().view(b, s_q, c)

    return x

After this processing, I found the image quality to be acceptable, but I'm unsure if there will be any problems with doing so.
